modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css

```diff
@@ -1,52 +1,297 @@
 .banner {
-	background-color: #f7f7f8;
-	border-radius: 1rem;
-	box-shadow: 0 0.3125rem 0.9375rem 0 rgba(19, 20, 31, 0.12);
-	gap: 1.5rem;
-	height: 7.5rem;
+	background: var(--neutral-01, #f7f7f8);
+	border-radius: 10px;
+	box-shadow: 0px 5px 15px 0px rgba(19, 20, 31, 0.12);
+	padding: 12px;
```

Comment:
Can we use bootstrap to set the padding and border-radius instead of using classes?